worker/src/core/loaders/message-worker/index.ts

```diff
@@ -248,6 +256,111 @@ const enqueueAndSend = async (): Promise<void> => {
   }
 }
 
+// to refactor to use orm
+const sendFinalizedNotification = (campaignId: number): void => {
```

Comment:
Is it OK to refactor the code here? 
- I don't think using `void` here is correct since we are actually using the result returned?
- The nested if statements are also quite hard to reason about

I would suggest doing sth like this:

```ts
const results = await connection.query(...)
const { id: ..., campaign_name... } =  get(result,...)
```

Then to get rid of the nested if statements:
- if `campaignId` doesn't exist, I'd think we should throw an error right? So we can use a guard clause to get rid of that?
- the final `if` statement is fine I guess haha

Reply:
removed campaignId if checker.

we are not using the return value of sendFinalizedNotification; it is a self contained function. we use function chaining so that we only execute the next portion of code after the the query has been completed and retrieve the results.

Reply:
my apologies, I just realized I'm highlighting the wrong line here

I meant we shouldn't use `void connection.query(...)` — the return type of `void` is correct!